jbx-protocol/juice-interface
# Upgrade some deps

Upgrade:

- ethereum-block-by-date
- bnc-onboard
- @walletconnect/web3-provider
- axios

Lock deps:
- @types/ethereum-block-by-date
- bnc-notify

### How to verify

1. Run the following on `main`

```
yarn add @walletconnect/web3-provider@1.8.0 bnc-onboard@1.39.0 axios@0.27.2 ethereum-block-by-date@1.4.5 @types/ethereum-block-by-date@1.4.1 bnc-notify@1.9.1 --exact
```

2. Git diff with this branch head.

```
git diff d695a88a20ad11a871f0d17d5f7885964c8ea841
```

There should be no diff.
